.NET/Microsoft.Recognizers.Definitions.Common/Turkish/NumbersWithUnitDefinitions.cs

```diff
@@ -23,40 +23,37 @@ public static class NumbersWithUnitDefinitions
     {
       public static readonly Dictionary<string, string> AgeSuffixList = new Dictionary<string, string>
         {
-            { @"Year", @"yaş|yaşında|yaşındayım|yaşındasın|yaşındayken|yaşından|yıllık" },
-            { @"Month", @"aylık|aylıkken|aydan" },
+            { @"Year", @"yaş|yaşında|yaşındayım|yaşındayken|yaşından|yıllık" },
+            { @"Month", @"aylık|aydan|aylıkken|ay" },
             { @"Week", @"haftalık|haftalıkken" },
             { @"Day", @"günlük" }
         };
       public static readonly Dictionary<string, string> AreaSuffixList = new Dictionary<string, string>
         {
-            { @"Kilometrekare", @"kilometre kare|kilometrekare|km2|km^2|km²" },
-            { @"Hektometrekare", @"hektometre kare|hektometrekare|hm2|hm^2|hm²" },
-            { @"Dekametrekare", @"dekametre kare|dekametrekare|dam2|dam^2|dam²" },
-            { @"Metrekare", @"metre kare|metrekare|m2|m^2|m²" },
-            { @"Desimetrekare", @"desimetre kare|desimetrekare|dm2|dm^2|dm²" },
-            { @"Santimetrekare", @"santimetre kare|santimetrekare|cm2|cm^2|cm²" },
-            { @"Milimetrekare", @"milimetre kare|milimetrekare|mm2|mm^2|mm²" },
-            { @"Mikrometrekare", @"mikrometre kare|mikrometrekare|µm2|µm^2|µm²" },
-            { @"Nanometrekare", @"nanometre kare|nanometrekare|nm2|nm^2|nm²" },
-            { @"İnçkare", @"inç kare|inçkare|in2|in^2|in²" },
-            { @"Fitkare", @"fit kare|fitkare|ft2|ft^2|ft²" },
-            { @"Milkare", @"mil kare|milkare|mi2|mi^2|mi²" },
-            { @"Yardakare", @"yarda kare|yardakare|yd2|yd^2|yd²" },
-            { @"Ayakkare", @"ayakkare" },
-            { @"Akre", @"akre|ac" },
+            { @"Square kilometer", @"kilometrekare|km2|km^2|km²" },
+            { @"Square hectometer", @"hektometrekare|hm2|hm^2|hm²" },
+            { @"Square decameter", @"dekametrekare|dam2|dam^2|dam²" },
+            { @"Square meter", @"metrekaredir|metrekare|m2|m^2|m²" },
+            { @"Square decimeter", @"desimetrekare|dm2|dm^2|dm²" },
+            { @"Square centimeter", @"santimetrekare|cm2|cm^2|cm²" },
+            { @"Square millimeter", @"milimetrekare|mm2|mm^2|mm²" },
+            { @"Square inch", @"inç kare|inçkare|in2|in^2|in²" },
+            { @"Square foot", @"fit kare|fitkare|ft2|ft^2|ft²|ayakkare" },
+            { @"Square mile", @"mil kare|milkare|mi2|mi^2|mi²" },
+            { @"Square yard", @"yarda kare|yardakare|yd2|yd^2|yd²" },
+            { @"Acre", @"akre|ac|dönümlük|dönüm" },
             { @"Ar", @"ar|a" },
             { @"Dekar", @"dekar|daa" },
-            { @"Hektar", @"hektar|ha" },
-            { @"Dönüm", @"dönüm" }
+            { @"Hektar", @"hektar|ha" }
         };
       public static readonly IList<string> AmbiguousAreaUnitList = new List<string>
         {
             @"fitkare",
             @"ar",
             @"dönüm",
             @"ha",
-            @"a"
+            @"a",
+            @"daa"
```

Comment:
Out of curiosity, what's "daa" ambiguous with?

Reply:
some company names are abbreviated as "daa"